common/data_refinery_common/management/commands/programs.tsv

```diff
@@ -0,0 +1,7 @@
+# progam_name	program_version	command
```

Comment:
This is an overly simple file for demo only. Obviously more programs should be added later.